tx-pool/src/component/container.rs

```diff
@@ -275,10 +275,22 @@ impl SortedTxMap {
         TxLink::get_descendants(&self.links, tx_short_id)
     }
 
-    /// return sorted keys
-    pub fn sorted_keys(&self) -> impl Iterator<Item = &AncestorsScoreSortKey> {
+    /// return keys sorted by tx fee rate
+    pub fn keys_sorted_by_fee(&self) -> impl Iterator<Item = &AncestorsScoreSortKey> {
         self.sorted_index.iter().rev()
     }
+
+    /// sorted keys by tx fee rate and transaction relation
+    pub fn keys_sorted_by_fee_and_relation(&self) -> Vec<&AncestorsScoreSortKey> {
```

Comment:
This function is only used when reorg removes entries from gap-pool/proposed-pool.
As I understand, we just expect that the returned entries are sorted by relation.
Hence I suggest renaming this function to `keys_sorted_by_relation`.

Reply:
Not only by relation.

Reply:
> Not only by relation.

Why we need the returned entries sorted by both fee and relation?

Reply:
After check it again, I think the only relation is no problem. I will update it.

Reply:
keep it as before.